src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -68,7 +68,7 @@ public static IEnumerable<RegexPattern> HighConfidenceSecurityModelsIterator()
         new AzureIotDeviceIdentifiableKey(),
         new AzureIotDeviceProvisioningIdentifiableKey(),
         new AzureStorageAccountIdentifiableKey(),
-        new AzureCosmosDbIdentifiableKey(),
+        new AzureCosmosDBIdentifiableKey(),
```

Comment:
>DB [](http://example.com/codeflow?start=23&length=2)

.net design guidelines specify `DB` here as casing. this is correct in our other scan tools. bringing this name into conformance.